lib/runtime-c-api/tests/runtime_c_api_tests.rs

```diff
@@ -6,7 +6,7 @@ fn test_c_api() {
 
     run_command("cmake", project_tests_dir, Some("."));
     run_command("make", project_tests_dir, Some("-Wdev -Werror=dev"));
-    run_command("make", project_tests_dir, Some("test"));
+    run_command("make", project_tests_dir, Some("test VERBOSE=1"));
```

Comment:
Is this an intentional change or debug code?

Reply:
It's intentional :)